test/fixtures/functionMsk/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "functionMsk",
+  "version": "1.0.0",
+  "description": "",
+  "main": "core.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
I think we can remove all those properties (`npm` doesn't require this meta data to exist for package installation)